docs/_ug/HowToUseUserGuide.md

```diff
@@ -20,3 +20,13 @@ If you are an experienced user, you can refer to the [Command Summary](#command-
 If you are stuck, refer to the section on [Troubleshooting](#troubleshooting) or [FAQ](#faq).
 
 You can also refer to the [Glossary](#glossary) for definitions of commonly used terms in FoodRem.
+
```

Comment:
Should we put a

```markdown
#### Admonishments
<!-- or maybe ##### -->

Admonishments are...
```

here?

Reply:
Good point, will add it

Reply:
I will replace Admonishments with FoodRem Icons, as Admonishments might be too technical